src/dlgTriggerEditor.cpp

```diff
@@ -650,15 +517,12 @@ void dlgTriggerEditor::slot_viewStatsAction()
 
 void dlgTriggerEditor::slot_viewErrorsAction()
```

Comment:
This is a bit weird - but see for yourself - there's a white rectangle that shows up if the errors view is of a certain width:

![peek 2017-03-26 11-16](https://cloud.githubusercontent.com/assets/110988/24330015/caf55b7e-1215-11e7-9d11-95ead678431e.gif)